vars/README.md

```diff
@@ -2164,6 +2164,16 @@ def i = randomNumber()
 def i = randomNumber(min: 1, max: 99)
 ```
 
+## randomString
+Generate a random string (alphanumeric and dash are allowed but not ending with dash_
```

Comment:
Can we add unit tests demonstrating this behaviour (even with higher number of iterations)?

```groovy
@Test  void test_not_ending_in_dash() throws Exception {
    for int i = 0; i < 100; i++ {
        def ret = script.call(size: 8)
        printCallStack()
        assertTrue(ret.charAt(7) != '_')
    }
}

Reply:
I'm refactoring the code to test only the string manipulation

Reply:
See https://github.com/elastic/apm-pipeline-library/pull/1147/commits/952dc7ec8e08d4e14781b0996bdc5873d00deb5b